apps/web/src/pages/Pools/EditPoolPage/components/UpdatePublishedProcessDialog/UpdatePublishedProcessDialog.tsx

```diff
@@ -36,7 +36,9 @@ const UpdatePublishedProcessDialog_Fragment = graphql(/* GraphQL */ `
   }
 `);
 
-export type FormValues = Partial<UpdatePublishedPoolInput>;
+export type FormValues = Partial<UpdatePublishedPoolInput> & {
+  changeJustification: UpdatePublishedPoolInput["changeJustification"];
+};
```

Comment:
I think this can be simplified?
```
export type FormValues = UpdatePublishedPoolInput;
```

Reply:
Oh yeah, idk what I was thinking here :stuck_out_tongue: Updated in [simplify form value type](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10348/commits/653ac6963099378628f86bd4b2dd78f6fc99ac81)